### PR TITLE
fix an obvious typo in loadSafePoint

### PR DESCRIPTION
**Pull request by zhiqiangxu**
### What problem does this PR solve? 

Fixed a typo in loadSafePoint

### What is changed and how it works?
changed a single variable name.

Side effects

NO

### CODE REVIEW

File: store/tikv/safepoint.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func saveSafePoint(kv SafePointKV, key string, t uint64) error {
 }
 
 func loadSafePoint(kv SafePointKV, key string) (uint64, error) {
```
Comment (jackysp) [inline store/tikv/safepoint.go]:
```suggestion
func loadSafePoint(kv SafePointKV) (uint64, error) {
```
how about? And `saveSafePoint` may have the same issue?

Comment (zhiqiangxu) [reply store/tikv/safepoint.go]: Good point, I've eliminated the `key` parameter altogether!

**Comment by disksing** [comment]
@zhiqiangxu Thanks for your contribution! 😊

**Comment by zhiqiangxu** [comment]
@disksing Is this file maintained now? https://github.com/pingcap/tidb/blob/master/CONTRIBUTORS

**Comment by disksing** [comment]
Good question... We will discuss if we want to update it or delete it. Will let you know after we decide it.

**Comment by zhiqiangxu** [comment]
@disksing OK

**Comment by disksing** [comment]
Hi @zhiqiangxu , the CONTRIBUTORS file will be deprecated and we will maintain the list in pingcap/community: https://github.com/pingcap/community/blob/master/contributors.md#tidb-contributors
We are still working on keep that list updated automatically.

**Comment by zhiqiangxu** [comment]
@disksing That's great !

**Pull request merged**